runtime/doc/lua.txt

```diff
@@ -3375,6 +3375,15 @@ vim.json.encode({obj}, {opts})                             *vim.json.encode()*
       • {opts}  (`table<string,any>?`) Options table with keys:
                 • escape_slash: (boolean) (default false) Escape slash
                   characters "/" in string values.
+                • pretty: (table) Table with keys:
+                  • enable: (boolean) (default false) Whether to enable
+                    pretty-formatting.
+                  • indent: (string) (default " ") String used for each
```

Comment:
This ate a single space during document generation (`" "`, but should be `"  "`). Maybe needs escaping with `` ` ``.